paths/comment_reactions/create.yaml

```diff
@@ -43,14 +49,14 @@ x-code-samples:
     curl "https://api.phrase.com/v2/projects/:project_id/keys/:key_id/comments/:comment_id/reactions/" \
       -u USERNAME_OR_ACCESS_TOKEN \
       -X DELETE \
-      -d '{"branch":"my-feature-branch"}' \
+      -d '{"branch":"my-feature-branch", "emoji": "👍"}' \
       -H 'Content-Type: application/json'
 - lang: CLI v2
   source: |-
     phrase reactions create \
     --project_id <project_id> \
     --key_id <key_id> \
     --comment_id <comment_id> \
-    --branch my-feature-branch \
+    --data '{"branch":"my-feature-branch", "emoji": "👍"]}' \
```

Comment:
interesting, does this mean that branch can't be specified with a dedicated switch?

Reply:
@jablan I'm exactly wondering about this too. In some places we have --branch param and in some just the --data with branch inside 🤔 What do you think? How should we do this so it will definitely work?

Reply:
IMO it would be the best if CLI could have parameters specified through the dedicated switches, and translate that to JSON payload to the API, but not sure if that works and how.